### ISSUE TITLE
My rewards not crossing 0.

### ISSUE THREAD

**Issue by ipsec** [danijar/dreamerv3#46]
Hi Danijar,

Is more a doub then an issue.
My custom environment reach reward 10 in others models/algorithms (like muzero). But, when using dreamer (v2 or v3) I get this behavior (the reward going to zero). I would love to use dreamerv3.

Do you already had this result before? Maybe need I some hyperparameters changes?

<img width="367" alt="image" src="https://user-images.githubusercontent.com/79345/232579909-45df84c2-78cc-4904-a551-cf51e85030ae.png">

I'm using this config and my observation is a image 64x64.

```
config = config.update(dreamerv3.configs['small'])
  config = config.update({
      'logdir': '~/logdir/run1',
      'run.train_ratio': 32,
      'run.log_every': 30,  # Seconds
      'batch_size': 8,
      'jax.prealloc': False,
      'encoder.mlp_keys': '$^',
      'decoder.mlp_keys': '$^',
      'encoder.cnn_keys': 'image',
      'decoder.cnn_keys': 'image',
      'jax.platform': 'cpu',
  })
```

This is a report when training:

```
────────────────────────────────────────────────────────────────────────────── Step 381545 ───────────────────────────────────────────────────────────────────────────────
train/action_mag 9 / train/action_max 9 / train/action_mean 7.97 / train/action_min 0 / train/action_std 0.42 / train/actor_opt_actor_opt_grad_overflow 0 /
train/actor_opt_actor_opt_grad_scale 1e4 / train/actor_opt_grad_norm 3.1e-6 / train/actor_opt_grad_steps 2.4e4 / train/actor_opt_loss -1356.35 / train/adv_mag 12.31 /
train/adv_max 0.88 / train/adv_mean -0.07 / train/adv_min -12.31 / train/adv_std 0.79 / train/cont_avg 0.99 / train/cont_loss_mean 1.3e-7 / train/cont_loss_std 2.9e-7 /
train/cont_neg_acc 1 / train/cont_neg_loss 8.6e-7 / train/cont_pos_acc 1 / train/cont_pos_loss 1.2e-7 / train/cont_pred 0.99 / train/cont_rate 0.99 / train/dyn_loss_mean
1.43 / train/dyn_loss_std 2.75 / train/extr_critic_critic_opt_critic_opt_grad_overflow 0 / train/extr_critic_critic_opt_critic_opt_grad_scale 1e4 /
train/extr_critic_critic_opt_grad_norm 0.09 / train/extr_critic_critic_opt_grad_steps 2.4e4 / train/extr_critic_critic_opt_loss 1.2e4 / train/extr_critic_mag 4.86 /
train/extr_critic_max -5.8e-3 / train/extr_critic_mean -0.26 / train/extr_critic_min -4.86 / train/extr_critic_std 0.27 / train/extr_return_normed_mag 12.03 /
train/extr_return_normed_max 0.52 / train/extr_return_normed_mean 0.19 / train/extr_return_normed_min -12.03 / train/extr_return_normed_std 0.85 / train/extr_return_rate
0.06 / train/extr_return_raw_mag 12.55 / train/extr_return_raw_max -1.6e-4 / train/extr_return_raw_mean -0.34 / train/extr_return_raw_min -12.55 /
train/extr_return_raw_std 0.85 / train/extr_reward_mag 5.37 / train/extr_reward_max -6.1e-6 / train/extr_reward_mean -0.02 / train/extr_reward_min -5.37 /
train/extr_reward_std 0.28 / train/image_loss_mean 0.06 / train/image_loss_std 0.17 / train/model_loss_mean 0.97 / train/model_loss_std 1.93 / train/model_opt_grad_norm
16.26 / train/model_opt_grad_steps 2.4e4 / train/model_opt_loss 9693.77 / train/model_opt_model_opt_grad_overflow 0 / train/model_opt_model_opt_grad_scale 1e4 /
train/policy_entropy_mag 0.08 / train/policy_entropy_max 0.08 / train/policy_entropy_mean 0.07 / train/policy_entropy_min 0.07 / train/policy_entropy_std 8.1e-5 /
train/policy_logprob_mag 6.91 / train/policy_logprob_max -9e-3 / train/policy_logprob_mean -0.07 / train/policy_logprob_min -6.91 / train/policy_logprob_std 0.65 /
train/policy_randomness_mag 0.03 / train/policy_randomness_max 0.03 / train/policy_randomness_mean 0.03 / train/policy_randomness_min 0.03 / train/policy_randomness_std
3.5e-5 / train/post_ent_mag 54.24 / train/post_ent_max 54.24 / train/post_ent_mean 11.24 / train/post_ent_min 6.25 / train/post_ent_std 7.23 / train/prior_ent_mag 61.4 /
train/prior_ent_max 61.4 / train/prior_ent_mean 13.41 / train/prior_ent_min 7.81 / train/prior_ent_std 8.98 / train/rep_loss_mean 1.43 / train/rep_loss_std 2.75 /
train/reward_avg -0.04 / train/reward_loss_mean 0.05 / train/reward_loss_std 0.46 / train/reward_max_data 5 / train/reward_max_pred 4.67 / train/reward_neg_acc 1 /
train/reward_neg_loss 0.05 / train/reward_pos_acc nan / train/reward_pos_loss nan / train/reward_pred -0.03 / train/reward_rate 0 / replay/size 3.8e5 / replay/inserts 32
/ replay/samples 16 / replay/insert_wait_avg 2.9e-6 / replay/insert_wait_frac 1 / replay/sample_wait_avg 1.5e-6 / replay/sample_wait_frac 0.75 / timer/duration 53.88 /
timer/env.step_count 32 / timer/env.step_total 0.02 / timer/env.step_frac 3.8e-4 / timer/env.step_avg 6.4e-4 / timer/env.step_min 3.9e-4 / timer/env.step_max 1.8e-3 /
timer/replay.add_count 32 / timer/replay.add_total 3.6e-3 / timer/replay.add_frac 6.6e-5 / timer/replay.add_avg 1.1e-4 / timer/replay.add_min 6.4e-5 /
timer/replay.add_max 4.4e-4 / timer/logger.write_count 1 / timer/logger.write_total 8e-3 / timer/logger.write_frac 1.5e-4 / timer/logger.write_avg 8e-3 /
timer/logger.write_min 8e-3 / timer/logger.write_max 8e-3 / timer/checkpoint.save_count 0 / timer/checkpoint.save_total 0 / timer/checkpoint.save_frac 0 /
timer/agent.save_count 0 / timer/agent.save_total 0 / timer/agent.save_frac 0 / timer/replay.save_count 0 / timer/replay.save_total 0 / timer/replay.save_frac 0 /
timer/agent.policy_count 32 / timer/agent.policy_total 0.35 / timer/agent.policy_frac 6.4e-3 / timer/agent.policy_avg 0.01 / timer/agent.policy_min 9e-3 /
timer/agent.policy_max 0.03 / timer/dataset_count 2 / timer/dataset_total 2.4e-4 / timer/dataset_frac 4.5e-6 / timer/dataset_avg 1.2e-4 / timer/dataset_min 9.2e-5 /
timer/dataset_max 1.5e-4 / timer/agent.train_count 2 / timer/agent.train_total 49.07 / timer/agent.train_frac 0.91 / timer/agent.train_avg 24.54 / timer/agent.train_min
24.4 / timer/agent.train_max 24.68 / timer/agent.report_count 1 / timer/agent.report_total 4.42 / timer/agent.report_frac 0.08 / timer/agent.report_avg 4.42 /
timer/agent.report_min 4.42 / timer/agent.report_max 4.42 / fps 0.59

Episode has 67 steps and return -3.6.
```

**Comment by danijar** [comment]
What environment?
